angular/projects/admin/src/app/admin/theme-template/theme-template-dialog-list-template/theme-template-dialog-list-template.component.html

```diff
@@ -0,0 +1,14 @@
+<h1 mat-dialog-title>List templates</h1>
+<div mat-dialog-content>
```

Comment:
It would be good if there was a text saying _"copy a template code to clipboard"_ to clarify that the code won't be inserted to the template by itself, but instead copied to the clipboard.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/135/commits/8b21e226816540f5504fd7f4dd2185afed706402